TheraLang.Web/ClientApp/src/njson-typings.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module "*.json" {
+    const value: any;
+    export default value;
+  }
```

Comment:
Do we need this?

Reply:
No, it's just a trash, I created this when was testing news output

Reply:
Remove?

Reply:
Yes, I removed, and also did some fixes you described below